libpod/runtime.go

```diff
@@ -691,24 +691,35 @@ func (r *Runtime) Info() ([]define.InfoData, error) {
 	}
 	info = append(info, define.InfoData{Type: "store", Data: storeInfo})
 
-	reg, err := sysreg.GetRegistries()
-	if err != nil {
-		return nil, errors.Wrapf(err, "error getting registries")
-	}
 	registries := make(map[string]interface{})
-	registries["search"] = reg
-
-	ireg, err := sysreg.GetInsecureRegistries()
+	regs, err := sysreg.GetRegistries()
 	if err != nil {
 		return nil, errors.Wrapf(err, "error getting registries")
 	}
-	registries["insecure"] = ireg
-
-	breg, err := sysreg.GetBlockedRegistries()
-	if err != nil {
-		return nil, errors.Wrapf(err, "error getting registries")
+	if len(regs) > 0 {
```

Comment:
`GetRegistries()` returns a slice of unqualified-search registries which is not an indicator for the config format. In fact, there is no indicator in the API at all to know if the parsed config file was in v1 or v2 format. `pkg/registries` is fully based on `containers/image/pkg/sysregistriesv2` which can parse both, v1 and v2, config files.

I think that the `else` branch is what we should be using here.

Reply:
Ok. So should I revert to my previous version and add the search table in addition to the registries data?

Reply:
Very good point! Yes, we need to add the search registries as well. To have a consistent naming, I would name the field `unqualified-search-registries` (see https://github.com/containers/image/blob/master/pkg/sysregistriesv2/system_registries_v2.go#L139). Thanks!

Reply:
Something like this:
```
registries:
  bad.docker.io:
    Blocked: false
    Insecure: true
    Location: bad.docker.io
    MirrorByDigestOnly: false
    Mirrors: []
    Prefix: bad.docker.io
  block.docker.io:
    Blocked: true
    Insecure: false
    Location: block.docker.io
    MirrorByDigestOnly: false
    Mirrors: []
    Prefix: block.docker.io
  search:
  - docker.io
  - registry.fedoraproject.org
  - registry.access.redhat.com
  - registry.centos.org
  - quay.io
```

Reply:
```
registries:                                                                                    
  docker.io:                                         
    Blocked: true                  
    Insecure: true           
    Location: docker.io      
    MirrorByDigestOnly: false
    Mirrors:               
    - Insecure: true                              
      Location: example2.io/example/ubi8-minimal                                               
    Prefix: docker.io         
  redhat.com:                      
    Blocked: false     
    Insecure: false
    Location: registry.access.redhat.com/ubi8
    MirrorByDigestOnly: true                             
    Mirrors:      
    - Insecure: false
      Location: example.io/example/ubi8-minimal
    - Insecure: true   
      Location: example3.io/example/ubi8-minimal
    Prefix: redhat.com  
```

Reply:
That looks good, now we need something similar in Buildah...